core/src/kvs/tx.rs

```diff
@@ -360,16 +365,16 @@ impl Transaction {
 	/// Retrieve all ROOT level users in a datastore.
 	#[instrument(level = "trace", target = "surrealdb::core::kvs::tx", skip(self))]
 	pub async fn all_root_users(&self) -> Result<Arc<[DefineUserStatement]>, Error> {
-		let key = crate::key::root::us::prefix();
-		let res = self.cache.get_value_or_guard_async(&key).await;
-		match res {
-			Ok(val) => val,
-			Err(cache) => {
+		let qey = cache::tx::Lookup::Rus;
```

Comment:
Is there a reason we call all these `qey` instead of `key`?

Reply:
Because `key` is used lower down as the actual storage engine key (`Vec<u8>`). Also, in quick_cache, the key which can be an owned or reference, is often called a `Q` or `qey` (equivalent key) so `qey` made sense.